tests/docker/run.sh

```diff
@@ -0,0 +1,28 @@
+TOPSRCDIR=$(cd ../..; printf %s "$PWD")
+MKFILEDIR=$PWD
+USERID=$(id -u)
+GROUPID=$(id -g)
+DOCKERFILE=Dockerfile
+
+NAME=navtest:$(git describe --tags)-$DOCKERFILE
+
+
+case $1 in
+
+    build)
+        docker build -t $NAME -f $DOCKERFILE $MKFILEDIR
+        ;;
+    buildnocache)
+        docker build --no-cache -t $NAME -f $DOCKERFILE $MKFILEDIR
+        ;;
+    check)
+        docker run -t -u $USERID:$GROUPID -v /$TOPSRCDIR:/source --tmpfs //var/lib/postgresql $NAME //source/tests/docker/test.sh
+        ;;
+    lint)
+        docker run -t -u $USERID:$GROUPID -v /$TOPSRCDIR:/source --tmpfs //var/lib/postgresql $NAME tox -e pylint
+        ;;
+    shell)
+        docker run -ti --rm -u $USERID:$GROUPID -v /$TOPSRCDIR:/source --tmpfs //var/lib/postgresql $NAME //bin/bash
+        ;;
+
+esac
```

Comment:
Github complains that the file does not end with a newline. On *nix text-files always end with a newline and odd things can happen if they don't.

Reply:
I added it to speed things up.

Reply:
Thanks @hmpf !